core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -477,6 +477,68 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
     }
   }
 
+  test("createDirectory") {
```

Comment:
Apache Spark community recommend to have a test name prefix with JIRA id. In this case,
```
- test("createDirectory") {
+ test("SPARK-35907: createDirectory") {
```